subsys/bluetooth/host/settings.h

```diff
@@ -10,6 +10,20 @@
 /* Base64-encoded string buffer size of in_size bytes */
 #define BT_SETTINGS_SIZE(in_size) ((((((in_size) - 1) / 3) * 4) + 4) + 1)
 
+/** @brief Convert a u8_t value into decimal string representation
```

Comment:
We don't generally use doxygen tags for any internal APIs since we don't want them to appear in any public documentation (which could risk people trying to use them from their apps)

Reply:
Will fix.
Agree, the code is not prettier, but we save a lot of code space with this change.